pkg/core/gas_price.go

```diff
@@ -0,0 +1,106 @@
+package core
+
+import (
+	"github.com/CityOfZion/neo-go/pkg/smartcontract"
+	"github.com/CityOfZion/neo-go/pkg/util"
+	"github.com/CityOfZion/neo-go/pkg/vm"
+	"github.com/CityOfZion/neo-go/pkg/vm/opcode"
+)
+
+// interopGasRatio is a multiplier by which a number returned from price getter
```

Comment:
nit: it will be nice to understand why do we need it at all, i.e. business meaning of interopGasRatio